### PR TITLE
ADD tests

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

**Comment by florian-dacosta** [comment]
https://github.com/akretion/server-tools/blob/9-external-file-location/external_file_location/tests/common.py

**Comment by angelmoya** [comment]
Thank you, I fixed last commit

File: attachment_base_synchronize/tests/test_attachment_base_synchronize.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    def test_attachment_metadata(self):
+        """Test run_attachment_metadata_scheduler to ensure set state to done
+        """
+        self.assertEqual(
+            self.attachment.state,
+            'pending'
+        )
+        self.ir_attachment_metadata.run_attachment_metadata_scheduler()
+        self.env.invalidate_all()
+        with openerp.registry(self.env.cr.dbname).cursor() as new_cr:
```
Comment (florian-dacosta) [inline attachment_base_synchronize/tests/test_attachment_base_synchronize.py:34]: Why do you have to open a new cursor here?
In the test of the other module (the link I sent previously), I think it is not needed

Comment (angelmoya) [reply attachment_base_synchronize/tests/test_attachment_base_synchronize.py:34]: I try to do without registry, and use invalidate_all to clean cache, but state still return 'pending', so with registry I am sure to read from database.

**Pull request merged**